ml-agents/mlagents/trainers/policy/policy.py

```diff
@@ -29,22 +29,28 @@ def __init__(
         condition_sigma_on_obs: bool = True,
     ):
         self.behavior_spec = behavior_spec
+        self.action_spec = behavior_spec.action_spec
```

Comment:
Why do we need self.behavior_spec and self.action_spec ? Seems like self.behavior_spec is enough

Reply:
there are a bunch of places where it gets accessed below but if we are fine with having self.behavior_spec.action_spec.x in a bunch of places thats ok with me